packages/devtools_extensions/example/foo/packages/foo_devtools_extension/lib/src/service_extension_example.dart

```diff
@@ -0,0 +1,235 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
```

Comment:
Consider adding some documentation at the top of this file describing what's going on in this file at a high level. Including references to service extension documentation would also be good.